Service/feature.logic.js

```diff
@@ -0,0 +1,18 @@
+const Util = require("./utils");
+const MongoStorage = require('../db/mongo.storage');
+const {ExperimentStorage} = require('../repositories/experiment.repository');
+
+
+const featureCheckAttributes =(endUserReq, experiment) =>{
+
+
+    const {ON, OFF} = experiment.variants_ff;
+    if (Util.checkAttributes(endUserReq, experiment))
+        return {"ON": ON};
+    else
+        return {"OFF": OFF};s
```

Comment:
so the logic here is if the attributes match we show on, and otherwise we show off?

Reply:
if the attributes match - then show the feature "ON"
else
"OFF"